nix/.stack.nix/default.nix

```diff
@@ -50,6 +50,7 @@
         cardano-prelude = ./cardano-prelude.nix;
         cardano-prelude-test = ./cardano-prelude-test.nix;
         cardano-crypto = ./cardano-crypto.nix;
+        cardano-crypto = ./cardano-crypto.nix;
```

Comment:
this duplicate `cardano-crypto` is breaking hydra (and by extension bors)

Reply:
Thanks for looking into this!